modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2SearchMembers.testcase

```diff
@@ -0,0 +1,230 @@
+
+			KeyPress(
+				locator1 = "CP2TeamMembers#SEARCH_TM",
+				value1 = "\ENTER");
+
+			AssertTextEquals(
+				locator1 = "CP2TeamMembers#ERROR_MESSAGE",
+				value1 = "No team members found with this search criteria.");
+		}
+	}
+
+	@description = "Automation ID: LPS-150834 | Test Summary: Verify if in the text box search it filters by email"
+	@priority = "4"
+	test VerifyTheSearchFiltersByEmail {
+		property portal.upstream = "quarantine";
+
+		// "Reason: Not able to reproduce scenario without OKTA workaround."
+
+		task ("Adding a user to the project") {
+			JSONUser.addUser(
+				userEmailAddress = "test.user@liferay.com",
+				userFirstName = "User",
+				userLastName = "2",
+				userScreenName = "Testing");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "test.user@liferay.com");
+
+			Accounts.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Test Account Portal");
+
+			Accounts.assignUsers(
+				accountName = "Test Account Portal",
+				assigneeName = "User 2");
+		}
+
+		task ("Go to CP2.0") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Login in Okta") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project") {
+			CustomerPortal.enterOnProject(
+				status = "Future",
+				title = "Test Account Portal");
+		}
+
+		task ("Go to the team members page") {
+			CP2NavigationMenu.goToTeamMembers();
+		}
+
+		task ("Search for the user's first name and affirm that the search worked") {
+			CP2TeamMembers.searchCP(search = "test.user@liferay.com");
+
+			KeyPress(
+				locator1 = "CP2TeamMembers#SEARCH_TM",
+				value1 = "\ENTER");
+
+			AssertElementPresent(
+				locator1 = "CP2TeamMembers#SEARCH_RESULT",
+				result = "1 result for test.user@liferay.com");
+		}
+
+		task ("Delete user created") {
+			JSONUser.deleteUserByEmailAddress(userEmailAddress = "test.user@liferay.com");
+		}
+	}
+
+	@description = "Automation ID: LPS-150832 | Test Summary: Verify if the text box search filters by first and last name"
+	@priority = "5"
+	test VerifyTheSearchFiltersByFirstAndLastName {
+		property portal.upstream = "quarantine";
+
+		// "Reason: Not able to reproduce scenario without OKTA workaround."
+
+		task ("Adding a user to the project") {
+			JSONUser.addUser(
+				userEmailAddress = "test.user@liferay.com",
+				userFirstName = "User",
+				userLastName = "2",
+				userScreenName = "Testing");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "test.user@liferay.com");
+
+			Accounts.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Test Account Portal");
+
+			Accounts.assignUsers(
+				accountName = "Test Account Portal",
+				assigneeName = "User 2");
+		}
```

Comment:
@luanakarla , since this task is added to all tests, we could add it into the setUp block. The test VerifyTheErrorMessageWhenNoResultsAreFound could use another inexistent value to guarantee the message appears.

Reply:
Done.